README.md

```diff
@@ -274,7 +274,9 @@ sign(asset, manifest);
 
 #### Local signing
 
-If you have a signing certificate and key, you can sign locally using a local signer; for example:
+If you have a signing certificate and key, you can sign locally using a local signer.  This is fine during development, but don't do it production because it exposes these sensitive files to potential attackers. Instead use a hardware security module (HSM) and optionally a Key Management Service (KMS) to access them; for example as show in the [C2PA Python Example](https://github.com/contentauth/c2pa-python-example).
```

Comment:
@crandmck Amazon's KMS is a multi-tenant HSM managed by them. I am not sure we'd say an HSM and KMS. I think we'd want all your PRs to update `Instead use a hardware security module (HSM) and optionally a Key Management Service (KMS) to access them;`

If I'm not mistaken, all the dev examples have the code passing the private key directly to the SDK. So I am thinking we could say something like `Instead use the SDK's external signer functionality to ensure no local systems have access to the key. The external signer should use solutions such as a Key Management Service (KMS) or hardware security module (HSM). You can see an example of using KMS in the C2PA Pyton Example.`

What do you think @mauricefisher64 @gpeacock @andrewhalle ? Whatever we say here we would want to repeat in the other PRs with this same verbiage.

Reply:
I'm no expert, but a bunch of Googling indicates that KMS and HSMs are orthogonal--KMS is the software and HSM is obviously a hardware solution, and they can be used together.  Cf
- https://www.jisasoftech.com/how-hardware-security-module-and-key-management-system-interact-in-enterprises/
- https://www.reddit.com/r/crypto/comments/n0ggbl/hsm_vs_kms/
> ... a KMS is just software that's used for managing keys in a networked environment. A KMS can be connected to a HSM for extra security.